md/script/evalCodeBlocks.ts

```diff
@@ -61,5 +69,9 @@ if (import.meta.main) {
     ? [[search, _replace]] as [string, string][]
     : undefined;
 
-  await evalCodeBlocks(filePath, replace);
+  const results = await evalCodeBlocks(filePath, replace);
+
+  results.forEach((result) => {
+    if (!(result instanceof Error) && !result.success) Deno.exit(1);
```

Comment:
Tested it in CI and it works 👍 

<img width="402" alt="Screen Shot 2023-05-06 at 3 01 01 PM" src="https://user-images.githubusercontent.com/47710574/236648235-bdb6bd69-be88-41ef-b20e-5c46c4060e5a.png">

<img width="1141" alt="Screen Shot 2023-05-06 at 3 01 39 PM" src="https://user-images.githubusercontent.com/47710574/236648244-d07fcb00-6b9b-48fb-90bc-68642b0c941e.png">